src/logger.js

```diff
@@ -1,33 +1,44 @@
 'use strict';
 const config = require('./config');
 const bunyan = require('bunyan');
-const Logger = require('le_node');
-
-const LOG_LEVEL = 'LOG_LEVEL';
+const bunyanLogentries = require('bunyan-logentries');
 const childLoggers = [];
 
 function getLogger(callingModule) {
+  let logLevel = config.get('LOG_LEVEL') || 'info';
   let callingFileName;
   if (callingModule) {
-    callingFileName = callingModule.filename.split('/').pop();
+    callingFileName = callingModule.filename.split('/').pop() || 'general';
```

Comment:
if there is no ``callingModule`` the logger will have ``name: undefined`` - why move `` || 'general'`` from line 15 ?